vowpalwabbit/oaa.cc

```diff
@@ -148,7 +148,7 @@ void predict(oaa& o, LEARNER::single_learner& base, example& ec)
       float sum_prob = 0;
       for (uint32_t i = 0; i < o.k; i++)
       {
-        ec.pred.scalars[i] = 1.f / (1.f + correctedExp(-o.pred[i].scalar));
+        ec.pred.scalars[i] = o.pred[i].scalar;
```

Comment:
Just to confirm: the combination of this and line 241 results in no change in test cases, yes?